scripts/gitlab.rb

```diff
@@ -1,3 +1,8 @@
+pages_external_url 'http://127.0.0.1:5051'
+pages_nginx['redirect_http_to_https'] = false
+pages_nginx['ssl_certificate'] = "/etc/gitlab/ssl/gitlab-registry.pem"

```

Comment:
I kinda dislike the naming of the re-used cert files here - since it's not only for the registry anymore. 

Also, is it configured properly for the pages to actually work? Regarding subdomains?

Reply:
@timofurrer I was thinking about these too but wanted to focus on the problem (the feature development). Tests in https://github.com/gitlabhq/terraform-provider-gitlab/pull/1026 pass with these settings in GitPod.

Reply:
Sure - I've created a follow up issue here:

* https://github.com/gitlabhq/terraform-provider-gitlab/issues/1042

Reply:
Feel free to merge once the tests passed 🎉